### DISCUSSION TITLE
Is there a way to combine the new inverted colors feature with the themed minor mode?

### DISCUSSION THREAD

**Discussion by skyler544** [vedang/pdf-tools#185] [General / FAQ]
Nice work on the inverted color view; it's a great step in the direction of having consistently readable dark-mode for PDFs in Emacs. 

If I understand what I'm seeing from the source code however, it seems like it's not possible to both keep hue and use the colors from the active theme? 

It would be very nice to have that feature; it's one of the few reasons why I still fall back to viewing PDFs in Zathura.

**Comment by vedang** [comment]
Short answer: I'm working on it! 

It isn't a complicated change, I think, and I do plan on adding support for inverted colors with the themed minor mode. I'm also considering making the themed minor mode the default. I just haven't gotten around to it yet. 

I will convert this discussion into an issue so that I can push changes against it when I get around to it.

**Comment by 88-percent** [comment]
This is fantastic news. Greatly appreciated.
